tfjs-backend-webgpu/src/backend_webgpu.ts

```diff
@@ -285,10 +285,11 @@ export class WebGPUBackend extends KernelBackend {
     const staging = this.acquireBuffer(
         info.bufferInfo.byteSize,
         GPUBufferUsage.COPY_DST | GPUBufferUsage.MAP_READ);
-    const encoder = this.device.createCommandEncoder();
-    encoder.copyBufferToBuffer(
+    if (!this.currentCommandEncoder) {
+      this.currentCommandEncoder = this.device.createCommandEncoder();
+    }
```

Comment:
optional nit: These three lines, used 4 times, could be wrapped in a `getOrCreateCommandEncoder()` method.

Reply:
Done. Use `ensureCommandEncoderReady` so that we don't need to return an encoder.